dinhducbkdn2004/Tintuc
# Add login function with password hashing and token handling

### Context
- Implement login functionality with security improvements (hashed passwords and token handling).

### Changes
- Added `PasswordHasher.java` to hash and update passwords in the database.
- Updated build path for the new library.
- Added `useToken` for managing authentication tokens.

### How to Test
1. Run `PasswordHasher.java` to update passwords in the database.
2. Verify login functionality works with hashed passwords.
3. Check token generation and validation.

### Notes
- Ensure the new library is added to the project build path.
